### PR TITLE
Sprite for an empty sentry box

### PR DESCRIPTION
**Pull request by Sanuki357**
## About The Pull Request

This pull adds a sprite for an empty sentry box.
When a sentry box is emptied, it will change its sprite to the new sprite which indicates that the box is empty.

The new sprite is imported into **TerraGov-Marine-Corps\icons\Marine\marine-weapons.dmi** where the original sentry box is stored.
The code for the detecting and switching to the empty sprite is borrowed from the M40 FLDP flare pack in **\code\game\objects\items\storage\boxes.dm**.
## Why It's Good For The Game

It was not easy to tell if a sentry box was empty or not, which made it annoying having to check if it is empty or not before tossing it out of a LZ or TAD. This is especially problematic since some people leave the ammo for the sentry inside the box, or leave it outside of the box, exacerbating the annoying need to check before disposing.
By giving it a visual cue, it is much easier to tell if the box is safe to be disposed or not.
I hope this brings a ease of mind to engineers, and they can have a better Engineer Gaming experience.
## Changelog
:cl:
imageadd: added sprite for an empty sentry box to make them distinguishable visually
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/sentries.dm [hunk 1]
```diff
@@ -54,6 +54,11 @@
 	new /obj/item/weapon/gun/sentry/big_sentry(src)
 	new /obj/item/ammo_magazine/sentry(src)
 
+/obj/item/storage/box/sentry/update_icon()
```
Comment (TiviPlus) [inline code/modules/projectiles/guns/sentries.dm:57]:
```suggestion
/obj/item/storage/box/sentry/update_icon_state()
```

**Review by TiviPlus — APPROVED** [review]
rest lgtm

**Review by TiviPlus — APPROVED** [review]
Tell kuro to merge

**Comment by Sanuki357** [comment]
> Tell kuro to merge

Gotcha. On Discord I presume?

**Pull request merged**